src/addons/model.js

```diff
@@ -1,19 +1,20 @@
 /**
  * @fileoverview A general modelling class used by the Presenter's
  * getModel function.
+ * @flow
  */
 
-import { set, Emitter } from '../microcosm'
+import { type Microcosm, Emitter, set } from '../microcosm'
 
-function isObservable(binding) {
+function isObservable(binding: *): boolean {
   return binding && typeof binding.subscribe === 'function'
 }
 
-function isCallable(binding) {
+function isCallable(binding: *): boolean {
```

Comment:
Using * means "let Flow infer it" This is super neat because you get really fantastic output from Flow while still being a little permissive about "This could literally accept anything"